src/components/NavBar/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { Colors, Styles } from '../../constants';
+import { MainMenu, IconProps } from './MainMenu';
+import { SbNavLink, SbNavlinkProps } from '../SbNavLink';
+import { SearchBar } from '../SearchBar';
+import { HelpCircle, Menu } from 'react-feather';
+
+/*tslint:disable: no-require-imports no-var-requires */
+const sbLogo = require('@sbac/sbac-ui-kit/src/images/SmarterBalanced-Logo.png');
+
+export interface NavBarProps {
+  links?: SbNavlinkProps[];
+  siteName: string;
+  mainContentId: string;
+}
+
+export const Logo = (): JSX.Element =>
+  <div className="nav-titleGroup-item">
+    <a
+      rel="noopener noreferrer"
+      target="_blank"
+      href="http://www.smarterbalanced.org/"
+      title="Smarter Balanced Home"
+    >
+      <img alt="Smarter Balanced Logo" src={sbLogo} />
```

Comment:
For some reason TS lint is complaining about this line of code Please fix and I will review.